packages/account/src/Components/poi/poi-country-selector/poi-country-selector.tsx

```diff
@@ -138,7 +153,7 @@ const CountrySelector = ({
                         <Button
                             className='proof-of-identity__submit-button'
                             type='submit'
-                            onClick={handleSubmit}
+                            onClick={() => handleSubmit()}
```

Comment:
please check if we can revert this or not?
it should work IMO

Reply:
Not working that way because of the onClick type!